### ISSUE TITLE
The importance of this app, vs NVAPI Limitations - Bug or Feature?

### ISSUE THREAD

**Issue by xcasxcursex** [Orbmu2k/nvidiaProfileInspector#46]
TLDR Recent driver broke stuff? We need this stuff! :( Has it been reported upstream? Bug or feature?

A recent commit https://github.com/Orbmu2k/nvidiaProfileInspector/commit/c2f71c5d1ea586fe7204ab486b4b66b73ed7272c Removed several settings and according to the description it is to remove errors, and according to a post I read elsewhere, the errors are caused by NVAPI not allowing these settings to be changed in newer driver versions.

> Since R445.75 the NVAPI does not allow either to read or write the following settings anymore. So i removed them because of errors.
> Code:
> 
> GSYNC - Indicator Overlay                0x10029538
> Ultra Low Latency                        0x10835000
> Sharpening - Denoising Factor            0x002ED8CE
> Sharpening Filter                        0x00598928
> Sharpening Value                         0x002ED8CD
> Optimize for Compute Performance         0x10C158AD
> Ultra Low Latency CPL State              0x0005F543

This is quite concerning for me, as the DCH driver model is to become standard, Nvidia have stated they will eventually phase out the standard driver model in favour of DCH, and DCH requires the control panel to be distributed exclusively by the Microsoft Store, which is not available to me - and many other users - this is not a small proportion of the userbase, educational and corporate versions of windows either do not even have the store installed, or have it locked down. 

Accordingly, nvidiaProfileInspector was a hope that we may still be able to control Nvidia cards. **This is a very important app**, especially in the future of DCH-only drivers.

It is concerning to see that now we have some features which cannot be controlled by the app because they cannot be interfaced via the driver API. I wonder if this is just a small accident and the driver has a bug which could be reported and fixed upstream, or if it is intentional to remove the ability to control the card by means other than the nvidia control panel (which is often unavailable).

By the way, I have contacted Nvidia directly regarding the inability to install the control panel in a corporate environment and their official response was "We'll get back to you", which they have not, even after 3 months... So I guess they don't care much :/ So, for now I have to hope for this app to keep working. This recent change is a little concerning :(
